src/System.Runtime.Extensions/tests/System/BitConverterArray.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Tests
+{
+    public class BitConverterArray : BitConverterBase
+    {
+        public override void ConvertFromBool(bool boolean, int expected)
+        {
+            Assert.Equal(expected, BitConverter.GetBytes(boolean)[0]);
```

Comment:
This one should be like the others and compare against a byte[].  Otherwise, the test would still pass even if BitConverter.GetBytes output a longer array than it should have, as long as the first byte was correct.